antora-ui-camel/src/js/vendor/algoliasearch.bundle.js

```diff
@@ -6,10 +6,13 @@
   window.addEventListener('load', () => {
     const client = algoliasearch('BH4D9OD16A', '16e3a9155a136e4962dc4c206f8278bd')
     const index = client.initIndex('apache_camel')
-    const search = document.querySelector('#search')
+    const condition = window.innerWidth < 1275
+    const search = condition ? document.querySelector('.search-small') : document.querySelector('.search')
```

Comment:
I'd prefer if we had a CSS only solution for this, not sure if there's need for two inputs for search. This way we make the implementation much more complex.

Reply:
I don't know why it seems complex 'cause the ternary operator does make the code readable. And the reason that two searches are included is just for breakpoint differentiation is 'cause if we were to include the single search input that is present within the nav, it will be more complex to position it the way we want in the design ( I will give it a try though ). Also, in my opinion, the use of two different div for the search design for two breakpoints seems much readable and it's easier to differentiate them using CSS as well.